### ISSUE TITLE
Conflict to @typescript-eslint/member-ordering

### ISSUE THREAD

**Issue by davidNHK** [infctr/eslint-plugin-typescript-sort-keys#30]
I like this plugin idea which will auto-fix the member ordering inside interface

but seem it would conflict to 
[member-order](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/member-ordering.md)

because this plugin don't care the member type 

E.G. given below interface

```typescript
interface Demo {
   b: string
   a() : void
}
```

it would complain error in above interface because a is after b.

But if fix it by move a before b, `member-ordering` would complain method should after attribute.

Beside turn off either one, did it any suggestion can keep both running ?

**Comment by infctr** [comment]
Yes, seems this would require signature type aware order sorting and/or allow the rule leverage `@typescript-eslint/member-ordering`

**Comment by davidnghk01** [comment]
Any hints on how should i did the code change ? i can help for open PR for that

**Comment by infctr** [comment]
Doesn't this do the work https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/member-ordering.md#sorting-alphabetically-within-member-groups?

**Comment by infctr** [comment]
I see. Well interface member type signature could be extracted from the node. Then the rule should accept an array of types as options that are going to be used for sorting just the way `member-ordering` does so that they're fully compatible, however I assume this is going to be a huge effort

**Comment by manojdcoder** [comment]
Would be nice to have this resolved, facing the same issue.

**Comment by nelson6e65** [comment]
There is a way to ignore methods and only sort keys?

**Comment by shellscape** [comment]
@infctr seems like @nelson6e65 landed on the right fix here. what's the chance of getting this fixed?
